aeneas/src/vst/Parser.v3

```diff
@@ -207,10 +233,18 @@ component Parser {
 				q++;
 			}
 			var id = extractIdent<void>(p, q);
-			if (id.kwClass != 0) kwError(p, id.name);
-			list = List.new(id, list);
+			var name = id.name.image;
+			var result: VstRepHint;
+			if (Strings.equal(name, "boxed")) result = VstRepHint.Boxed;
+			else if (Strings.equal(name, "unboxed")) result = VstRepHint.Unboxed;
+			else if (Strings.equal(name, "packing")) {
```

Comment:
Packing is a keyword now; let's introduced `KC_PACKING` to allow it (but not other keywords) here.